08-regression2.Rmd

```diff
@@ -118,20 +124,18 @@ small_plot +
 
 Simple linear regression chooses the straight line of best fit by choosing
 the line that minimizes the **average** vertical distance between itself and
-each of the observed data points. From the lines shown above, that is the blue
-line. What exactly do we mean by the vertical distance between the predicted
-values (which fall along the line of best fit) and the observed data points?
-We illustrate these distances in the plot below with a red line:
+each of the observed data points in the training data. Figure \@ref(fig:08-verticalDistToMin) illustrates 
+these vertical distances as red lines. Finally, to assess the predictive 
```

Comment:
I think we should be very clear here about the formula being used to calculate the **average** vertical distance between itself and each of the observed data points. **Also, it is the sum of the squared distances (not the average vertical distances), correct? And so we need to change this?**

Reply:
yeah def squared dists, fixed